### DISCUSSION TITLE
When 1.4 officially will be released?

### DISCUSSION THREAD

**Discussion by danieltomasz** [quarto-dev/quarto-cli#7765] [Q&A] [answered]
### Description

Just to know the time perspective - next quartier, next half year or hard to say?
I might use some features of quarto 1.4 in  a future project,  but it need to be installed/updated through the IT department and it will not be easy to convince anyone  about using beta version.

Also - is there a general cycle for releases or you release new version when you feel ready/it contains all planned features?

**Comment by eitsupi** [comment]
Looking at the GitHub milestone, it looks like 57% complete.
<https://github.com/quarto-dev/quarto-cli/milestone/9>
And, the number of issues registered for the 1.4 milestone seems to be almost double that of the 1.3 milestone at this time.

Considering the length of time since 1.3 was released, I fear that 1.4 will not be released for another 6 months or more.

**Reply by baptiste** [reply]
I remember seeing [this comment in passing – "targeting end of year"](https://mastodon.social/@scheidegger/111285813330917356), but that was a while ago (and not a promise!) so it may have been pushed back a little.

**Reply by cscheid** [reply, marked as answer]
> Considering the length of time since 1.3 was released, I fear that 1.4 will not be released for another 6 months or more.

I'll just say "we hope and expect to pleasantly surprise you". No promises, of course; @baptiste's comment is largely accurate on our current plans.
